install/image-references

```diff
@@ -14,6 +14,10 @@ spec:
     from:
       kind: DockerImage
       name: registry.svc.ci.openshift.org/openshift:openstack-machine-controllers
+  - name: openstack-machine-api-provider
```

Comment:
This string needs to match https://github.com/openshift/ocp-build-data/blob/b75c21e23e55142ea58b037d0d4c4e7878b33ba2/images/ose-machine-api-provider-openstack.yml#L25 once you strip the `openshift/ose-` prefix

Reply:
It should be good now thanks to https://github.com/openshift/ocp-build-data/pull/1275.

Reply:
The image is not yet available in the `4.10-art-latest` image stream though, I'll let you know when we can merge this.

Reply:
It's good to go, the image is there now:
```
$ oc describe imagestream 4.10-art-latest | grep openstack-machine-api-provider
openstack-machine-api-provider
```